common-util/hooks/useUpdateModuleDetails.ts

```diff
@@ -0,0 +1,58 @@
+import { cloneDeep } from 'lodash';
+import { useDispatch } from 'react-redux';
+
+import { getModuleDetails, proposeOrUpdatePost } from 'common-util/api';
+import { setModuleDetails, useAppSelector } from 'store/setup';
+import type { ModuleDetails } from 'store/types';
+
+export const useUpdateModuleDetails = () => {
+  const { moduleDetails, moduleDetailsAttributeId, isModuleDetailsLoading } = useAppSelector(
+    (state) => state.setup,
+  );
+  const dispatch = useDispatch();
+
+  const getUpdatedModuleDetailsAfterPostProposal = (
+    tweetDetails: ModuleDetails['scheduled_tweet']['tweets'][number],
+  ) => {
+    if (!moduleDetails) return;
+
+    const updatedModuleDetails = cloneDeep(moduleDetails);
```

Comment:
we can't rely on this, because the data is no longer fetched by intervals. can you please check my comment in the api endpoint re this

Reply:
ah right, i completely missed that.